client/src/App.css

```diff
@@ -0,0 +1,240 @@
+/********** global styles **********/
+
+* {
+    padding: none;
+    margin: none;
+    font-family: 'Fira Sans', sans-serif;
+}
+
+html, body {
+    height: 100%;
+    margin: 0;
+}
+
+/********** header styles **********/
+
+#headerLine {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    font-size: 40px;
+    background: linear-gradient(rgb(61, 56, 121), rgb(126, 121, 182));
+    height: 12vh;
+    /* font-family: 'Bellota Text', cursive; */
+    font-family: 'Gotu', sans-serif;
+    font-variant: small-caps;
+    font-weight: 700;
+}
+
+#langButtons {
+    display: flex;
+    align-items: flex-end;
+    justify-content: flex-end;
+    padding: 0 2rem 1rem 0;
+    height: 8vh;
+    background: linear-gradient( rgb(126, 121, 182), rgb(61, 56, 121))
+}
+
+/********** main content areas **********/
+
+#body {
+    display: flex;
+    justify-content: center;
+    padding: 2rem;
+    height: 60vh;
+    max-width: 95vw;
+    font-family: 'Fira Sans', sans-serif;
+    position: relative;
+    height: auto;
+    min-height: 100%
+}
+
+#contentContainer {
+    display: flex;
+    flex-direction: space-around;
+    padding: 2rem;
+    min-height: 100%;
+    min-width: 80vw;
+    overflow: auto;
+}
+
+#centeriece {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 2rem;
+    min-height: 100%;
+    max-width: 100%;
+    border: 2px solid red;
+    margin-top: 2rem;
+    text-align: center;
+}
+
+#image {
+    padding: 0 0 1rem 0;
+    max-height: 40vh;
+    display: flex;
+    justify-content: center;
+}
+
+#portrait {
+    max-height: 400px;
+    max-width: 250px;
+    display: block;
+    margin: 0 1rem;
+    border-radius: 5px;
+}
+
+#caption {
+    text-align: center;
+}
+
+#tagline {
+    font-size: 20px;
+    font-style: italic;
+    font-weight: 500;
+    margin: 3rem;
+}
+
+.rotated {
+    transform: rotate(90deg)
+}
+
+#pages {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    min-width: 65vw;
+    margin: auto;
+    height: auto;
+}
+
+#blankSpace {    
+    width: 100%;
+    padding-right: 4rem;
+}
+
+#navbar {
+    padding: 2rem;
+    max-width: 12vw;
+}
+
+/*********** sticky footer setup? I really don't know if this does anything **********/
+
+#pushFooter {
+    padding: 20px;
+    min-height: 100%;
+    margin: 0 auto -10vh;
+}
+
+.push {
+    height: auto;
+}
```

Comment:
This stuff is giving your footer a little bit of spacing, but isn't making it sticky. An easy way that you can check what each rule does, is by commenting out all of them and then un-commenting each one and watching the results.